kratos/mpi/tests/cpp_tests/utilities/test_parallel_fill_communicator.cpp

```diff
@@ -76,4 +76,59 @@ KRATOS_TEST_CASE_IN_SUITE(ParallelFillCommunicatorExecute, KratosMPICoreFastSuit
     }
 }
 
+KRATOS_TEST_CASE_IN_SUITE(ParallelFillCommunicatorExecuteTriangleMesh, KratosMPICoreFastSuite)
```

Comment:
does this work with any number of cores?

Reply:
Yes, because for more than 2 partitions it is always the same